cockatrice/src/filtertree.cpp

```diff
@@ -22,46 +22,51 @@ void FilterTreeBranch<T>::deleteAt(int i)
 template <class T>
 int FilterTreeBranch<T>::childIndex(const FilterTreeNode *node) const
 {
-    FilterTreeNode *unconst;
-    T downcasted;
-
-    unconst = const_cast<FilterTreeNode *>(node);
-    downcasted = dynamic_cast<T>(unconst);
-    if (downcasted == NULL)
-        return -1;
-    return childNodes.indexOf(downcasted);
+    FilterTreeNode *unconst = const_cast<FilterTreeNode *>(node);
+    T downcasted = dynamic_cast<T>(unconst);
+    return (downcasted) ? childNodes.indexOf(downcasted) : -1;
 }
 
 template <class T>
 FilterTreeBranch<T>::~FilterTreeBranch()
 {
     while (!childNodes.isEmpty())
+    {
```

Comment:
Can you adjust the style guide and remove the instruction to skip braces for 1-liner blocks?

Reply:
I'll get around to updating all the files this week

Reply:
It seems every new dev skips braces on 1-liners until they discover it leads to errors later. Glad to hear that's not the desired style.

Reply:
Went that way originally because it's very nice in scala. Of course scala is expression-oriented so a typical 'missing brace' code error will pretty much always make compiling fail instead of silently being weird. Doesn't make sense for C++